Resources/Prototypes/Reagents/Consumable/Food/condiments.yml

```diff
@@ -53,3 +53,30 @@
   desc: A salty soy-based flavoring.
 #  physicalDesc:
   color: saddlebrown
+
+- type: reagent
+  id: TableSalt
+  name: table salt
+  desc: Commonly known as salt, Sodium Chloride is often used to season food or kill borers instantly.
+  physicalDesc: grainy
+  color: "#a1000b"
+  boilingPoint: 1465.0
+  meltingPoint: 800.7
+  plantMetabolism:
+  - !type:PlantAdjustWater
+    amount: -3
+  - !type:PlantAdjustNutrition
+    amount: -0.3
+  - !type:PlantAdjustToxins
+    amount: 8
+  - !type:PlantAdjustWeeds
+    amount: -2
+  - !type:PlantAdjustPests
+    amount: -1
+  - !type:PlantAdjustHealth
+    amount: -2
+  metabolisms:
+    Food:
+      effects:
+      - !type:SatiateHunger
+        factor: 1
```

Comment:
I don't know about you, but I wouldn't be able to stave off hunger by just eating pure salt. At the very least it should probably make you thirsty....

Reply:
sure, i'll have it make you thirstier